corehq/apps/userreports/reports/view.py

```diff
@@ -398,6 +399,11 @@ def get_ajax(self, params):
         }
         if total_row is not None:
             json_response["total_row"] = total_row
+        if data_source.data_source.config.backend_id == UCR_LABORATORY_BACKEND:
+            compare_ucr_dbs.delay(
+                self.domain, self.report_config_id, self.filter_values,
+                sort_column, sort_order, params
+            )
```

Comment:
Not sure how many of these requests come in but I don't think we should do this for every request.